### PR TITLE
ignore unpacked package data files

### PR DESCRIPTION
**Pull request by eugenesvk**

### CODE REVIEW

**Comment by gerardroche** [comment]
Add those locally on your system. They're not relevant to the plugin.

You can set local ignore rules either globally or local to the specific repository. 

Ignore files locally specific to repository: `.git/info/exclude`

See https://www.git-scm.com/docs/gitignore

**Comment by eugenesvk** [comment]
They are relevant to the plugin since these are plugin files even though they're autogenerated by sublime

**Comment by gerardroche** [comment]
What is generating the files? How are you working on NeoVintageous locally? I updated the contributing doc with some information on how work on nv locally: https://github.com/NeoVintageous/NeoVintageous/blob/develop/.github/CONTRIBUTING.md. Whatever is creating those files should be necessary.

tldr; Install via a git repository directly into your Packages directory and disable PC auto-updates for it.

**Comment by eugenesvk** [comment]
Package Control generates those files, so they are not global, but package specific and thus should be ignored in the package itself (`.extracted-sublime-package` is unique to ST, so can be globally ignored, but not `package-metadata.json`)
Don't get it, what's the downside of ignoring more junk within the project?
